config/unicorn.rb

```diff
@@ -0,0 +1,23 @@
+# config/unicorn.rb
+worker_processes Integer(ENV["WEB_CONCURRENCY"] || 3)
+timeout 15
+preload_app true
+
+before_fork do |server, worker|
```

Comment:
Unused block argument - `server`. You can omit all the arguments if you don't care about them.<br>Unused block argument - `worker`. You can omit all the arguments if you don't care about them.